### DISCUSSION TITLE
Range Rover Stolen - still have active connections to incontrol

### DISCUSSION THREAD

**Discussion by mus247** [ardevd/jlrpy#98] [Q&A]
Hi All,

4 weeks ago my Range Rover Evoque was stolen outside my house. - thankfully the insurance paid me out - but I still want to have the last laugh and get back at them whatever way I can. 

I have a data sim card in the car and  to my suprise I still can access to incontrol as they haven't removed the sim. Unfortunately, I'm unable to track the location of the vehicle and they must have spoofed the address. But  I'm curious and looking for recommendations on how I can plot my revenge does anyone have any cool ideas?

**Comment by ardevd** [comment]
Sorry to hear about your car getting stolen. 

I'm curious how you are trying to track the car. Are you saying the location reported by the InControl API is inaccurate? 

As long as you have access to the InControl service for that vehicle you can do any number of things, including unlocking the doors, honking the horn and starting the climate and/or engine.
